mockupdb/__init__.py

```diff
@@ -1207,19 +1209,12 @@ def __init__(self, port=None, verbose=False,
         self._autoresponders = []
 
         if auto_ismaster is True:
-            self.autoresponds(Command('ismaster'),
+            self.autoresponds(CommandBase('ismaster'),
```

Comment:
Curious: Does PyMongo ever send isMaster over OP_MSG?

Reply:
Yes, the initial handshake uses OP_QUERY then if maxWireVersion>=6 all subsequent commands on the socket use OP_MSG.